lib/config/all.js

```diff
@@ -11,11 +11,10 @@ module.exports = {
   },
 
   parserOptions: {
-    ecmaVersion: 2017,
+    ecmaVersion: 2018,
```

Comment:
Part of the migration guide: [The experimentalObjectRestSpread option has been deprecated](https://eslint.org/docs/user-guide/migrating-to-5.0.0#-the-experimentalobjectrestspread-option-has-been-deprecated)